perceiver/data/audio/giantmidi_piano.py

```diff
@@ -0,0 +1,55 @@
+import functools
+import hashlib
+import os
+import shutil
+from pathlib import Path
+from typing import Any, Dict
+
+from perceiver.data.audio.symbolic import SymbolicAudioDataModule
+from perceiver.data.audio.utils import download_file, extract_file
+
+
+class GiantMidiPianoDataModule(SymbolicAudioDataModule):
+    def __init__(
+        self,
+        *args: Any,
+        dataset_uri: str = "https://perceiver-io-giant-midi.s3.amazonaws.com/giantmidi-piano.zip",
```

Comment:
A copy is now available at https://martin-krasser.com/perceiver/data/midi/giantmidi-piano.zip